reqwest-tracing/Cargo.toml

```
@@ -42,12 +42,12 @@ getrandom = { version = "0.2.0", features = ["js"] }
 tokio = { version = "1.0.0", features = ["macros"] }
 tracing_subscriber = { package = "tracing-subscriber", version = "0.3.0" }
 wiremock = "0.6.0"
-reqwest = { version = "0.12.0", features = ["rustls-tls"]}
-
+reqwest = { version = "0.12.0", features = ["rustls-tls"] }
```

Thanks for fixing the formatting here. Let's keep the newline though since it's nice to logically separate the opentelemetry dependencies from the others.

```suggestion
reqwest = { version = "0.12.0", features = ["rustls-tls"] }

```